### DISCUSSION TITLE
Catching Disinfluencies of language

### DISCUSSION THREAD

**Discussion by aldnoahh** [espnet/espnet#3580] [Q&A]
I have been trying catch disinfluencies (filler words, “um” or “uh”, extra pauses, etc.) of English language using different models (wav2vec, deepspeech) but have been unable to do so. Is there any way to catch them with without manual labelling them in the speech dataset??

**Comment by sw005320** [comment]
Such disfluencies often happen in spontaneous conversation and the model trained by read speech cannot well capture it. I recommend you use the model trained with the switchboard corpus and its variant.
